JASP-Desktop/po/jasp_nl.po

```diff
@@ -11791,15 +11791,15 @@ msgstr "Evalueer Passing"
 
 msgctxt "LDgamma|"
 msgid "Histogram vs. theoretical pdf"
-msgstr "Histogram vs. theoretische pdf (kansverdelingsfunctie)"
+msgstr "Histogram vs. theoretische pdf"
 
 msgctxt "LDgamma|"
 msgid "Q-Q plot"
 msgstr "Q-Q plot"
 
 msgctxt "LDgamma|"
 msgid "Empirical vs. theoretical cdf"
-msgstr "Empirisch vs. theoretisch cdf (cumulatieve verdelingsfunctie)"
+msgstr "Empirisch vs. theoretische cdf"
```

Comment:
Je gaat dit zoziezo replace-allen of niet :p

Reply:
Yes all